pkg/discovery/mock/mock.go

```diff
@@ -43,12 +40,12 @@ func (d *Discovery) Broadcasts() int {
 	return d.ctr
 }
 
-func (d *Discovery) AddresseeRecord(addressee swarm.Address) (overlay swarm.Address, addr ma.Multiaddr) {
+func (d *Discovery) AddresseeRecords(addressee swarm.Address) (peers []swarm.Address, exists bool) {
 	d.mtx.Lock()
 	defer d.mtx.Unlock()
 	rec, exists := d.records[addressee.String()]
 	if !exists {
-		return swarm.Address{}, nil
+		return []swarm.Address{}, false
 	}
-	return rec.Overlay, rec.Addr
+	return rec, true
```

Comment:
Hmm, I think it's not possible because it's `comma ok` idiom and compiler is complaining because there is not two parameters. But I can do this: 
``` peers, exists = d.records[addressee.String()]
	return ```

Reply:
Ah, yes, you are right, thanks.